denv

```diff
@@ -7,13 +7,16 @@ denv () {
             denv_enter $2 $3
         ;;
         r|restart)
-            denv_restart $2
+            denv_restart $2 $3
         ;;
         s|reset)
-            denv_reset $2
+            denv_reset $2 $3
+        ;;
+        r|remove)
```

Comment:
There's a conflict on shortcut "r"

Reply:
Also, what's the difference with denv_exit?

Reply:
I changed `remove` to `delete`.

`denv_exit` just stop the container (you sill see them as stopped with "docker ps -a")
`denv_delete` will also delete them and delete the associated volumes

Reply:
I don't see the different use case for `delete` and `exit`. How about merging them?

Reply:
That's work for me.